src/sidebar/tab.js

```diff
@@ -169,7 +169,12 @@ SideTab.prototype = {
     const imgTest = document.createElement("img");
     imgTest.src = favIconUrl;
     imgTest.onerror = () => {
-      this._resetIcon();
+      // https://bugzilla.mozilla.org/show_bug.cgi?id=1462948
+      if (favIconUrl === "chrome://mozapps/skin/extensions/extensionGeneric-16.svg") {
```

Comment:
How about we do the detection at the beginning of `_setIcon` instead and set the right URL directly?

Reply:
Yes, that’s better!